sources/packages/web/src/services/ApplicationOfferingChangeRequestService.ts

```diff
@@ -110,6 +114,19 @@ export class ApplicationOfferingChangeRequestService {
     );
   }
 
+  /**
+   * Gets the Application Offering Change Request status.
+   * @param applicationOfferingChangeRequestId the application offering change request id.
+   * @returns the application offering change request status.
+   */
+  async getApplicationOfferingChangeRequestStatusById(
```

Comment:
better to avoid `ById` in web methods

Reply:
https://github.com/bcgov/SIMS/pull/2196#discussion_r1303506903

Reply:
@dheepak-aot @andrepestana-aot @guru-aot suggestions?